en/components/general/ignite-ui-licensing.md

```diff
@@ -152,24 +152,44 @@ The following information is on how to setup authentication to our private npm r
 Update the azure-pipelines.yml with the following steps:
 
 ```cmd
-npm config set @infragistics:registry $(npmRegistry)
-```
+pool:
+  vmImage: 'Ubuntu 16.04'
 
-```cmd
-npm config set always-auth true --scope:@infragistics
-```
+steps:
+
+- script: echo "@infragistics:registry=$(npmRegistry)" >> ~/.npmrc
```

Comment:
You can keep the old `npm config set` scripts, or would you like to streamline all guidance to `echo` into the `.npmrc` file? Also you define a variable group below, but you never set it in the script:

```
variables:
- group: Common-variables
```